src/kms.ts

```diff
@@ -338,17 +338,29 @@ export class KmsClient {
    * @param {Uint8Array} derBytes the DER certificate/private key as bytes
```

Comment:
Edit the Import phrase just above - remove 'or private key'

Reply:
done in a4855b7d4b9ca2562caa4bf2ac21b1153dd984cf